provider/kubernetes/kubernetes.go

```diff
@@ -50,6 +50,7 @@ type Provider struct {
 	Namespaces             Namespaces `description:"Kubernetes namespaces"`
 	LabelSelector          string     `description:"Kubernetes api label selector to use"`
 	lastConfiguration      safe.Safe
+	Redirect               string `description:"Kubernetes Redirect"`
```

Comment:
Similar to what @nmengin suggested from the @containous/docker team, I think we should clearly identify that this should be an `entrypointRedirect`